sub/info.md

```
@@ -1,7 +1,8 @@
+更新时间2025-03-20 23:15:57
 
-**总订阅: 181**
-**总节点: 30693**
-- vless节点: 10866
+**总订阅: 160**
+**总节点: 30497**
+- vless节点: 10163
+- hysteria2节点: 2
 
-**google ping有效节点: 1427**
+**google ping有效节点: 684**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)